funsize/tasks/funsize.yml

```diff
@@ -96,6 +96,8 @@ tasks:
         - index.funsize.v1.{{ branch }}.revision.{{ platform }}.{{ revision }}.{{ update_number }}.{{ chunk_name }}{% if subchunk %}_{{ subchunk }}{% endif %}.signing
         - index.funsize.v1.{{ branch }}.latest.{{ platform }}.{{ update_number }}.{{ chunk_name }}{% if subchunk %}_{{ subchunk }}{% endif %}.signing
       extra:
+        signing:
+          signature: {{ sign_task(stableSlugId("signing_task"), valid_for=8 * 3600) }}
```

Comment:
You might comment this with the assertion that funsize is making here.  Something like "assert that this signing task was created by the real funsize"